packages/-ember-data/package.json

```diff
@@ -73,13 +73,14 @@
     "ember-qunit": "*",
     "ember-qunit-assert-helpers": "*",
     "ember-resolver": "*",
-    "ember-source": "*",
```

Comment:
this shouldn't be necessary

Reply:
my bad, this was left over from running tests on a specific version.